models.py

```diff
@@ -354,6 +364,12 @@ class Chapter(models.Model):
     filename = models.CharField(
         max_length=255,
     )
+    license_information = models.TextField(
+        blank=True,
+        null=True,
+        help_text="Add copyright and/or license information here. "
+                  "If left blank will display book version of this field.",
+    )
```

Comment:
Is there a reason to not use submission.Licence here?

Reply:
Flexibility - book licenses aren't the same as articles.